app/src/test/java/nl/tudelft/bbw/APITest.java

```diff
@@ -46,14 +47,20 @@
      */
     @Before
     public final void setUp() throws HashException, BlockAlreadyExistsException {
+
```

Comment:
They are used to divide the parts of a method for better analysis. It's more of individual coding style.
But some unnecessary lines are removed.